README.md

```diff
@@ -1601,8 +1615,242 @@ p.createFeeRule(data: {
+})
+```
+
+
+### Payment Method 
+
+Instanitiate Payment Method using constructor that has been injected with Xendit keys
+
+```js
+const { PaymentMethod } = x;
+const r = new PaymentMethod();
+```
+
+Example: Create a Payment Method
+
+```js
+r.createPaymentMethod({
+      type: 'DIRECT_DEBIT',
+      reusability: 'ONE_TIME_USE',
+      customer_id: '16f72571-9b3a-43dc-b241-5b71f470202f',
+      country: 'ID',
+      direct_debit: {
+        channel_code: 'BRI',
+        channel_properties: {
+          mobile_number: '+6281299640904',
+          card_last_four: '8888',
+          card_expiry: '10/29',
+          email: 'dharma@xendit.co',
+        },
+      },
+}).then(({ id }) => {
+  console.log(`payment method created with ID: ${id}`);
+});
 ```
 
+
+#### Create payment method
+
+```ts
+r.createPaymentMethodV2(data: {
+  type: PaymentMethodV2Types;
+  reusability: PaymentMenthodV2Reusabilities;
+  reference_id?: string;
+  customer_id?: string;
+  country?: CreatePaymentMenthodV2Countries;
+  description?: string;
+  billing_information?: BillingInformationItems;
+  metadata?: object;
+  ewallet?: EwalletItems;
+  direct_debit?: DirectDebitItems;
+  card?: CardItems;
+  over_the_counter?: OverTheCounterItems;
+  virtual_account?: VirtualAccountItems;
+  qr_code?: QRISItems;
+  for_user_id?: string;
+  idempotency_key?: string;
+})
+```
+
+#### List payment methods
+
+```ts
+r.listPaymentMethodV2(data: {
+  payment_method_id?: string;
+  payment_method_type?: string;
+  channel_code?: string;
+  limit?: number;
+  after_id?: string;
+  before_id?: string;
+  for_user_id?: string;
+})
+```
+
+#### Get payment method details by ID
+
+```ts
+r.getPaymentMethodByIdV2(data: {
+    id: string;
+    for_user_id?: string;
+})
+```
+
+#### Authorize Payment method
+
+```ts
+r.authorizePaymentMethodV2(data: {
+    id: string;
+    auth_code: string;
+    for_user_id?: string;
+    idempotency_key?: string;
+})
+```
+
+#### Update Payment method
+
+```ts
+r.updatePaymentMethodV2(data: {
+    id: string;
+    reference_id?: string;
+    description?: string;
+    metadata?: object;
+    status?: object;
+    reusability?: PaymentMenthodV2Reusabilities;
+    over_the_counter?: UpdateOverTheCounterItems;
+    virtual_account?: UpdateVirtualAccountItems;
+    for_user_id?: string;
+})
+```
+
+#### Expire Payment method
+
+```ts
+r.expirePaymentMethodV2(data: {
+    id: string;
+    for_user_id?: string;
+    idempotency_key?: string;
+})
+```
+
+#### List payments by payment method id
+
+```ts
+r.listPaymentsByPaymentMethodIdV2(data: {
+    id: string;
+    payment_request_id?: string;
+    reference_id?: string;
+    status?: ListPaymentMethodV2StatusItems;
+    limit?: number;
+    after_id?: string;
+    before_id?: string;
+    created?: string;
+    updated?: string;
+    for_user_id?: string;
+})
+```
+
+
+Refer to [Xendit API Reference](https://developers.xendit.co/api-reference/#refunds) for more info about methods' parameters
```

Comment:
Is this supposed to link to https://developers.xendit.co/api-reference/#payment-methods ?

Reply:
yup my bad. changed to payment methods and added a link for payment request too